cmd/podman/common/completion.go

```diff
@@ -58,8 +58,9 @@ func setupContainerEngine(cmd *cobra.Command) (entities.ContainerEngine, error)
 	}
 	if !registry.IsRemote() {
 		_, noMoveProcess := cmd.Annotations[registry.NoMoveProcess]
+		cgroupMode := cmd.LocalFlags().Lookup("cgroups").Value.String()
```

Comment:
This can be called for many commands and you never check that the flag is defined like you do below in the other case. You need to do the same here otherwise the shell completion will be broken due the nil deref.